docs/understanding-tracking-design/introduction-to-tracking-design/best-practices/index.md

```diff
@@ -0,0 +1,248 @@
+Identifying and analysing the use case should be the first step before we design our tracking.
+This involves identifying the business outcomes that you want to achieve (e.g., acquire new customers, increase the number of signups, minimize abandoned carts)?
+It can often be answered by interviewing stakeholders who will consume the data in your organization.
+
+One of the ways to think about this is to sketch the reports that we want to have – the derived tables that the modeling process will extract from the raw events.
+Based on the reports, one can identify the entities, events, and SQL queries to be performed on top of them.
+
+:::tip Example – abandoned carts in e-commerce
+
+For example, if one of the use cases is to identify abandoned carts, we may want the following reports to explore the data:
+
+* What is the frequency of users abandoning carts for different segments of users?
+* What are the most commonly abandoned products in carts?
+* What leads users to return to the store and continue their purchase?
+
+We need to go deeper into specifying what exactly should the derived tables for these reports contain.
+However, we can already see a few reusable entities that we need to track: user, cart, product, page referrer.
+The reports can also tell us what events we'll need as interactions of these entities: add product to cart, checkout step, transaction, page view.
+
+:::
+
+## Make use of our out-of-the-box schemas
+
+Snowplow trackers and data models provide a rich set of events and entities to cover common use cases.
+These include:
+
+* user and session identification,
+* device and browser information,
+* e-commerce events and entities,
+* media playback events and entities,
+* error and performance tracking,
+* and much more.
+
+It is recommended to use and build on top of these data structures in order to save effort and make use of the existing Snowplow dbt packages and tooling.
+To get a comprehensive overview of the out-of-the-box provided schemas, [please refer to this page](/docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md).
+
+Our [data product accelerators](https://snowplow.io/data-product-accelerators/) are a great resource to develop your data product faster using out-of-the-box data structures based on best practices.
+
+## Choose your naming convention
+
+Having a common standard for how to name events, entities and properties can make it easier to understand and extend the tracking design.
+The following are our recommendations, but in general, it's more important to be consistent across your schemas than to follow these recommendations:
+
+* Use snake case for both the schema and properties. This will ensure that the names consistent with how the properties end up in the warehouse (for some warehouses, property names are converted to snake case regardless of how they are defined in the schemas). Avoid using hyphens to separate words, instead use underscores.
```

Comment:
```suggestion
* Use snake case for both the schema and properties. This will ensure that the names are consistent with how the properties end up in the warehouse (for some warehouses, property names are converted to snake case regardless of how they are defined in the schemas). Avoid using hyphens to separate words, instead use underscores.
```